torch_geometric/utils/coalesce.py

```diff
@@ -37,6 +37,31 @@ def coalesce(
 
     :rtype: :class:`LongTensor` if :attr:`edge_attr` is :obj:`None`, else
         (:class:`LongTensor`, :obj:`Tensor` or :obj:`List[Tensor]]`)
+
+    Example:
```

Comment:
```suggestion
    Examples::
```

Reply:
I found `Examples::` and `Examples:` are quite different in doc after complied, where the latter is boldfaced.
+ `Examples::`
<img width="707" alt="image" src="https://user-images.githubusercontent.com/39986668/189824347-a554996a-d309-46e5-b663-cb564b16b5f4.png">

+ `Examples:`
<img width="709" alt="image" src="https://user-images.githubusercontent.com/39986668/189824983-7fa3db64-bba5-413e-84f0-c878b77c2c30.png">


Which one would you prefer?

Reply:
Interesting. Looks like we need to modify the CSS for the `::` syntax. Let's keep it as `Examples:` then :(

Reply:
Got it.